srai/loaders/gtfs_loader.py

```diff
@@ -0,0 +1,161 @@
+"""
+GTFS Loader.
+
+This module contains GTFS loader which performs time aggregations from timetable data.
+It utilizes the `gtfs_kit` library [1]. It was originally created for the purpose of
+the gtfs2vec project [2].
+
+References:
+    [1] https://gitlab.com/mrcagney/gtfs_kit
+    [2] https://doi.org/10.1145/3486640.3491392
+
+"""
+
+from pathlib import Path
+from typing import TYPE_CHECKING
+
+import geopandas as gpd
+import pandas as pd
+from shapely.geometry import Point
+
+from srai.utils._optional import import_optional_dependencies
+from srai.utils.constants import WGS84_CRS
+
+if TYPE_CHECKING:  # pragma: no cover
+    from gtfs_kit import Feed
+
+
+class GTFSLoader:
+    """
+    GTFSLoader.
+
+    This loader is capable of reading GTFS feed and calculates time aggregations in 1H slots.
+
+    """
+
+    def __init__(self) -> None:
+        """Initialize GTFS loader."""
+        import_optional_dependencies(dependency_group="gtfs", modules=["gtfs_kit"])
+
+        self.time_resolution = "1H"
+
+    def load(
+        self,
+        gtfs_file: Path,
+        fail_on_validation_errors: bool = True,
+        skip_validation: bool = False,
+    ) -> gpd.GeoDataFrame:
+        """
+        Load GTFS feed and calculate time aggregations for stops.
+
+        Args:
+            gtfs_file (Path): Path to the GTFS feed.
+            fail_on_validation_errors (bool): Fail if GTFS feed is invalid.
```

Comment:
Maybe make it explicit here, that this is ignored if skip_validation is True?

Reply:
Fair point, I will add it then